portfolio/src/main/webapp/style.css

```diff
@@ -64,14 +80,40 @@ html {
     background-image: url("/images/head3.jpeg"); 
 }
 
-.jumbotron {
+#about-title {
+    font: 32px Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    text-align: center;
+    text-decoration: underline;
+}
+
+.about-body{
+    font: 16px/2 Arial, Helvetica, sans-serif;
```

Comment:
This is neat with the font shorthand, I haven't seen this before

Reply:
Yah, I actually learned it from the style guide.